src/phases/query/BooleanSimplifier.yml

```diff
@@ -0,0 +1,231 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-optimization"
+
+Description: |
```

Comment:
Could you make it clear in the comment the distinction between measuring the effectiveness of the simplifier (that is, how much it improved the final plan, so that the plan got faster) vs the efficiency of the simplifier (that is, how fast does the simplifier does its job). The latter would be tested via large predicates, and very small or empty datasets, while the former would be tested via various input predicates over reasonably large dataset.

Reply:
These benchmarks are not designed to test efficiency of the simplifier, only effectiveness, which is already stated in the comment. For example, these benchmarks test the simplifier's efficiency: https://github.com/10gen/mongo/blob/dd975024e8e11bf250c9a75310020bb33a689e5d/src/mongo/db/query/boolean_simplification/match_expression_bm.cpp#L175-L313